Trading/Mode/index_trading_mode/config/IndexTradingMode.json

```diff
@@ -0,0 +1,6 @@
+{
+    "required_strategies": [],
+    "refresh_interval": 7,
```

Comment:
Should we use 1 or 3 as default values as the open source bot will do nothing else and it seems to be better values than 7 days ?

Reply:
indeed